build/scripts/build-impl.xml

```diff
@@ -344,11 +343,26 @@
         <echo message="Creating exist.jar ..."/>
         <jar basedir="${build.classes}" compress="true" jarfile="exist.jar">
             <manifest>
-                <attribute name="Project-Name" value="${project.name}"/>
-                <attribute name="Project-Version" value="${project.version}"/>
-                <attribute name="Project-Build" value="${DSTAMP}"/>
-                <attribute name="Project-CodeName" value="${project.codename}"/>
-                <attribute name="Git-commit" value="${git.commit}"/>
+                <attribute name="Implementation-Title" value="${project.name} Core"/>
+                <attribute name="Implementation-Version" value="${project.version}"/>
+                <attribute name="Implementation-Vendor" value="The eXist Project"/>
```

Comment:
I took the `Implementation-Vendor` name from the name that we use in the LGPL header, as that is as close to anything that we have which actually defines an name for the open source organisation.